test/k8s-integration/main.go

```diff
@@ -287,9 +314,9 @@ func handle() error {
 		var err error = nil
 		switch *deploymentStrat {
 		case "gce":
-			err = clusterUpGCE(k8sDir, *gceZone, *numNodes, *imageType)
+			err = clusterUpGCE(k8sDir, *gceZone, *numNodes, testParams.imageType)
 		case "gke":
-			err = clusterUpGKE(*gceZone, *gceRegion, *numNodes, *imageType, *useGKEManagedDriver)
+			err = clusterUpGKE(*gceZone, *gceRegion, *numNodes, testParams.imageType, *useGKEManagedDriver)
```

Comment:
Let's use testParams.useGKEManagedDriver for uniformity? similar to line 386

Reply:
Oops, thanks, done. Also updated the deploymentStrategy.